app/views/users/_user.html.erb

```diff
@@ -16,7 +17,26 @@
 
   <p>
     <strong>Posts counter:</strong>
-    <%= user.posts_counter %>
+    <%= user.posts.count %>
   </p>
-
 </div>
+<%end%>
+
+
+<%#############################################%>
```

Comment:
- [ ] [OPTIONAL] Kindly remove all unused codes/comments in order to have a clean codebase.